ximea_ros2_cam/config/xiCam1_config.yaml

```diff
@@ -0,0 +1,109 @@
+ximea_cam1_node:
+  ros__parameters:
+    ####################
+    # Specific Configuration Parameters Go Here!
+    ####################
+  
+    serial_no: "UYGAS2411015"
```

Comment:
document each parameter

Reply:
done, in readme file I wrote the complete description of each parameter.

Reply:
Commit: https://github.com/Brazilian-Institute-of-Robotics/ximea_ROS2_driver/pull/1/commits/2bb6e0eae9117872aa357f1acf15b0cc064acba2